### DISCUSSION TITLE
Using KM distance in blocking_rules_to_generate_predictions parameter

### DISCUSSION THREAD

**Discussion by srrmtt** [moj-analytical-services/splink#2566] [Q&A]
Hi everyone,

may anyone tell me if is it possible to use a condition like  the distance in km between coordinates columns in the blocking_rules_to_generate_predictions parameter?

Thanks in advance

**Comment by RobinL** [comment]
You can, see [here](https://moj-analytical-services.github.io/splink/api_docs/blocking.html?h=customrule#splink.blocking_rule_library.CustomRule) but it's generally not recommended see [here](https://moj-analytical-services.github.io/splink/topic_guides/blocking/performance.html?h=#equi-join-conditions)

it might be ok if your data is small
